starlette/responses.py

```diff
@@ -41,7 +49,7 @@ def __init__(
         self.body = self.render(content)
         self.init_headers(headers)
 
-    def render(self, content: typing.Any) -> bytes:
+    def render(self, content: Content | None) -> bytes:
```

Comment:
`|` was introduced in 3.10 in

Reply:
We use `__future__.annotations`, and the pipeline passes. 👀